components/Cart/CartPage/CartItemsContainer.component.jsx

```diff
@@ -97,9 +97,9 @@ const CartItemsContainer = () => {
             />
             <div className="mt-4">
               <Link href="/kasse" passHref>
-                <button className="px-4 py-2 font-bold bg-white border border-gray-400 border-solid rounded hover:bg-gray-400">
+                <a className="px-4 py-2 font-bold bg-white border border-gray-400 border-solid rounded hover:bg-gray-400">
```

Comment:
Why is this no longer a button?

Reply:
passHref does not pass onto native HTML elements other than a tag.